packages/php-wasm/logger/src/lib/collectors/collect-window-errors.ts

```diff
@@ -0,0 +1,94 @@
+import { Logger } from '../logger';
+
+/**
+ * Log Windows errors.
+ *
+ * @param loggerInstance The logger instance
+ * @param ErrorEvent event
+ */
+const logWindowError = (loggerInstance: Logger, event: ErrorEvent) => {
```

Comment:
logWindowError made me thing "Oh, is window.error a property?". This would make it clearer:

```suggestion
const logWindowErrorEvent = (loggerInstance: Logger, event: ErrorEvent) => {
```